tensorly/regression/cp_regression.py

```diff
@@ -80,12 +82,13 @@ def fit(self, X, y):
         """
         rng = T.check_random_state(self.random_state)
 
-        # Initialise randomly the weights
+        # Initialise the weights randomly
         W = []
+
         for i in range(
             1, T.ndim(X)
         ):  # The first dimension of X is the number of samples
-            W.append(T.tensor(rng.randn(X.shape[i], self.weight_rank), **T.context(X)))
```

Comment:
Check that I resolved the merge conflict correctly here.

Reply:
Indeed, I definitely think I messed up the merge here.

Reply:
I fixed it. Everything is working fine 😀